pkg/proxy/proxier.go

```diff
@@ -315,16 +316,19 @@ type Proxier struct {
 // NewProxier returns a new Proxier given a LoadBalancer and an address on
 // which to listen.  Because of the iptables logic, It is assumed that there
 // is only a single Proxier active on a machine.
-func NewProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables.Interface) *Proxier {
+func NewProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables.Interface, hostIP net.IP) *Proxier {
 	if listenIP.Equal(localhostIPv4) || listenIP.Equal(localhostIPv6) {
 		glog.Errorf("Can't proxy only on localhost - iptables can't do it")
 		return nil
 	}
 
-	hostIP, err := util.ChooseHostInterface()
-	if err != nil {
-		glog.Errorf("Failed to select a host interface: %v", err)
-		return nil
+	if hostIP == nil {
```

Comment:
Not at all - works great!  I just figured it should be configurable.

I can just dump the flag if you'd prefer (until we need it?).